### PR TITLE
Fix REST API schema output

### PR DESCRIPTION
**Pull request by roborourke**
Schema properties should have been callbacks.

- [x] Updated changelog
- [x] Updated version number in `package.json` and `plugin.php` file with appropriate MAJOR.MINOR.PATCH change

### CODE REVIEW

**Review by shadyvb — CHANGES_REQUESTED** [review]
Looks good, however I don't think it covers all API controllers we have, eg: https://github.com/humanmade/workflows/blob/master/lib/events/new-editorial-comment.php#L301 ( and other too ).

File: lib/events/new-editorial-comment.php [hunk 1]
```diff
@@ -298,13 +298,20 @@ function assignees_api() {
...
+			return [
+				'$schema' => 'http://json-schema.org/draft-04/schema#',
+				'title' => 'Manage assigness for a post',
+				'type' => 'object',
+					'properties' => [
+					'id' => [
+						'type' => 'integer',
+						'required' => true,
+						'description' => __( 'The post ID to modify assignees on.', 'hm-workflows' ),
+					],
```
Comment (shadyvb) [inline lib/events/new-editorial-comment.php:311]: Got a missing bracket here
```suggestion
					'properties' => [
						'id' => [
							'type' => 'integer',
							'required' => true,
							'description' => __( 'The post ID to modify assignees on.', 'hm-workflows' ),
						],
					],
```

**Review by shadyvb — CHANGES_REQUESTED** [review]

File: lib/events/new-editorial-comment.php (hunk 1 above)
Comment (roborourke) [reply lib/events/new-editorial-comment.php:311]: Wasn't missing, just not properly indented

Comment (shadyvb) [reply lib/events/new-editorial-comment.php]: True, my bad.

**Review by shadyvb — APPROVED** [review]

**Pull request merged**